infrastructure/guardduty/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.62.1"
+      version = "~> 4.62.2"
```

Comment:
We need to make sure not to have any instances of these aws module versions included.

Reply:
Done. Setting everything to 4.62.0 and keeping there.